configs/config.ini

```diff
@@ -136,6 +141,11 @@ password = admin
 tenant = admin
 temp = /root/merge
 
+#"service_tenant" is Optional value which describe name for Tenant used by openstack services.
+#By default CloudFerry takes following value for service tenant: "service".
+#To check if the service_tenant value is different from default run: "keystone tenant-list | grep service" and "keystone tenant-get $tenant_id"
```

Comment:
Please keep comment lines no longer than 79 symbols, it's very difficult to read and compare files otherwise